sys/userlib/src/lib.rs

```diff
@@ -969,6 +969,13 @@ pub fn sys_irq_control(mask: u32, enable: bool) {
     }
 }
 
+#[inline(always)]
```

Comment:
Should this have a doc comment (perhaps reproducing what's in `syscalls.adoc`)?

Reply:
Probably, though note that none of the stubs in the Oxide userlib have comments. I'll add one to this one.